languageFiles/en.json

```diff
@@ -132,5 +132,7 @@
   "Closed Captions":"Closed Captions",
   "Full Screen": "Full Screen",
   "Exit Full Screen": "Exit Full Screen",
-  "More Options": "More Options"
+  "More Options": "More Options",
+  "Audio": "Audio",
+  "Subtitles": "Subtitles"
```

Comment:
We already have "Closed Captions". Can't we use that variable instead of "Subtitles" ?

Reply:
We need to this label from design